xdmod/install.sh

```diff
@@ -45,6 +45,9 @@ yum install -y https://tas-tools-ext-01.ccr.xdmod.org/9.0.0rc2/xdmod-9.0.0-0.2.r
                https://tas-tools-ext-01.ccr.xdmod.org/9.0.0rc2/xdmod-supremm-9.0.0-0.2.rc2.el7.noarch.rpm \
                https://github.com/ubccr/supremm/releases/download/1.4.0rc01/supremm-1.4.0-rc01.el7.x86_64.rpm
 
+# modify portal_settings.ini to enable CORS for OnDemand
```

Comment:
Make this look like the other comments to maintain consistency.  I think we might want to make it more readable as well

```
#------------------------
# Open XDMoD allows integration with applications running on different servers using CORS.
# For security, you must specify what those servers are.
#------------------------
```